test/functional/lcow_bench_test.go

```diff
@@ -0,0 +1,101 @@
+//go:build windows && functional
+// +build windows,functional
+
+package functional
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Microsoft/hcsshim/osversion"
+
+	"github.com/Microsoft/hcsshim/test/internal/require"
+	"github.com/Microsoft/hcsshim/test/internal/uvm"
+)
+
+func BenchmarkLCOW_UVM_Create(b *testing.B) {
+	requireFeatures(b, featureLCOW)
+	require.Build(b, osversion.RS5)
+
+	ctx := context.Background()
+
+	b.StopTimer()
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		opts := defaultLCOWOptions(b, b.Name())
+
+		b.StartTimer()
+		vm := uvm.CreateLCOW(ctx, b, opts)
+		b.StopTimer()
+
+		// vm.Close() hangs unless the vm was started
+		cleanup := uvm.Start(ctx, b, vm)
+		cleanup()
+	}
+}
+
+func BenchmarkLCOW_UVM_Start(b *testing.B) {
+	requireFeatures(b, featureLCOW)
+	require.Build(b, osversion.RS5)
+
+	ctx := context.Background()
+
+	b.StopTimer()
+	b.ResetTimer()
+	for i := 0; i < b.N; i++ {
+		vm := uvm.CreateLCOW(ctx, b, defaultLCOWOptions(b, b.Name()))
+
+		b.StartTimer()
+		if err := vm.Start(ctx); err != nil {
```

Comment:
why aren't we using the same 'uvm.Start' function here?

Reply:
it has a bunch of test-related overhead I dont want to profile
https://github.com/microsoft/hcsshim/blob/bf3434f03a044021100bf698d83fa88d3167ebc1/test/internal/uvm/uvm.go#L17-L28

Reply:
That's pretty negligible code though, I wouldn't expect any of it to cause a marked slowdown

Reply:
Thats true, I can change it back if you think it distracts unnecessarily?